consensus/polybft/consensus_runtime.go

```diff
@@ -205,9 +205,9 @@ func (c *consensusRuntime) OnBlockInserted(block *types.Block) {
 	// after the block has been written we reset the txpool so that the old transactions are removed
 	c.config.txPool.ResetWithHeaders(block.Header)
 
-	// handle commitment and bundles creation
-	if err := c.createCommitmentAndBundles(block.Transactions); err != nil {
-		c.logger.Error("failed to create commitments and bundles", "error", err)
+	// handle commitment and proofs creation
+	if err := c.getCommitmentFromTransactions(block.Transactions); err != nil {
+		c.logger.Error("on block inserted error", "err", err)
```

Comment:
Should we change this log message to be more specific?

For example, like this (although doesn't need to be exactly like this):
```suggestion
		c.logger.Error("failed to retrieve commitment from transactions, "err", err)
```